src/monitormanager.h

```diff
@@ -49,6 +49,7 @@ class MonitorManager : public IndexingObject<Monitor> {
     void unlock();
     std::string lock_number_changed();
 
+    int printStack(Output output);
```

Comment:
I don't think we've ever talked about it, but in many cases, we already use a `Command` suffix for methods like these. At times, I've really appreciated that clarification when reading the code, as it was more visible than noticing the fact that it has `Input` and/or `Output` as arguments. I think there is even a case where it's "necessary" in the sense that there is a non-command member `foo()` that does a thing and a `fooCommand()` that provides the command parsing interface for doing the same thing.

I'd suggest adopting that pattern for all command members.

Reply:
Sometimes I've tried to keep the name short where it's obvious that it's a command. I've renamed it to `stackCommand()` to make it obvious that this is the implementation of the command `stack`.

I agree with your suggestion to have the `…Command` suffix for all command member functions.